department-of-veterans-affairs/Virtual-Regional-Office
# Houli/circle ci setup

# To Test

Make a branch off of this branch. (This is to ensure that `.circleci/config.yml` is in your branch.)

Create a GitHub PR for your branch. If you don't have a PR for your branch, Circle CI won't run. (_Circle CI will run every time you make a commit to a branch that has a GitHub PR._)

Log into Circle CI. Navigate to https://app.circleci.com/pipelines/github/department-of-veterans-affairs/Virtual-Regional-Office

1. Make a commit that breaks the pytests. Circle CI should run. It should fail.

2. Make a commit that fixes the tests. Circle CI should run. It should pass.

Alternative to logging into Circle CI: Just look at your GitHub PR page, at the "checks" section near the merge button.

# Cleanup

Delete your PR and branch.
